security/acme-client/src/opnsense/mvc/app/models/OPNsense/AcmeClient/AcmeClient.xml

```diff
@@ -473,6 +473,7 @@
                         <dns_loopia>Loopia</dns_loopia>
                         <dns_lua>LuaDNS.com</dns_lua>
                         <dns_miab>MailinaBox</dns_miab>
+                        <dns_mythic>Mythic Beasts</dns_mythic>
```

Comment:
I think this will not work. The underlying code uses the value of "dns_service" and passes it to acme.sh:
https://github.com/opnsense/plugins/blob/ce1995a5852037ea9b342cde8099ebd659b3fa52/security/acme-client/src/opnsense/mvc/app/library/OPNsense/AcmeClient/LeValidation/Base.php#L82

However, the name of the acme.sh dnsapi is "mythic_beasts", not "mythic". So the name should be corrected in all places.

Reply:
Fixed :)